src/Dropdown.php

```diff
@@ -2255,6 +2259,24 @@ public static function showFromArray($name, array $elements, $options = [])
                 $output .= " data-track-changes=''";
             }
 
+            if ($param['aria_label'] !== '') {
+                $output .= " aria-label='" . $param['aria_label'] . "'";
+            }
+
+            if (!empty($param['add_select_attributes'])) {
+                if (is_array($param['add_select_attributes'])) {
```

Comment:
Accepting a string seems dangerous here, as we will not be able to handle its encoding properly.